fluentd/configs.d/user/fluent.conf

```diff
@@ -19,6 +19,7 @@
   @include configs.d/openshift/filter-pre-*.conf
   @include configs.d/openshift/filter-retag-journal.conf
   @include configs.d/openshift/filter-k8s-meta.conf
+  @include configs.d/openshift/filter-k8s-events.conf
```

Comment:
@wozniakjan @richm Is there a need to conditionally include this for cases where 'eventrouter' is not deployed?

Reply:
if eventrouter is conditionally deployed, then yes, we need support in openshift-ansible, in the fluentd daemonset, in the mux dc, and in fluentd/run.sh, which will make this a 0 length file if eventrouter is not used.